sdk/storage/AzureStorageBlob/Source/DataStructures/URLSessionTransferManager.swift

```diff
@@ -110,6 +105,28 @@ public final class URLSessionTransferManager: NSObject, TransferManager, URLSess
         return transfers[index]
     }
 
+    public func transfer(forName name: String, type: TransferType? = nil) -> Transfer? {
```

Comment:
Unsure if we want to ultimately keep this. I was working on adding unique IDs for each transfer, which the customer can use to retrieve a Transfer object by its ID. Retrieving by file name or part thereof seems like it could become a source of bugs.

Reply:
Yep, I agree. :)